### PR TITLE
Habib/deriv me url redirection

### PR DESCRIPTION
**Pull request by ShuvoHabib**
Changes:

- change all links pointing to app.deriv.com , community.deriv.com, blog.deriv.com smarttrader.deriv.com, to app.deriv.me, community.deriv.me, blog.deriv.me , smarttrader.deriv.me (check localized-link.js)
- oauth links are pointing to oauth.deriv.me
- change the appId to - TBD

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [x] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by mustofa-binary — DISMISSED** [review]
LGTM

**Review by yashim-deriv — DISMISSED** [review]

File: src/common/constants.js [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
+const domain_url =
+    isBrowser() && supported_domains.includes(window.location.hostname)
+    ? window.location.hostname
+    : 'deriv.com'
+
+export const deriv_app_url = `https://app.${domain_url}`
+export const deriv_bot_app_url = `https://app.${domain_url}/bot`
+export const deriv_dp2p_app_url = `https://app.${domain_url}/cashier/p2p`
+export const smarttrader_url = `https://smarttrader.${domain_url}`
+export const blog_url = `https://blog.${domain_url}`
```
Comment (mustofa-binary) [inline src/common/constants.js:13]: we are not supporting community and blog for deriv.me, please revert the change for those two links

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

File: src/common/constants.js [hunk 2]
```diff
@@ -1,22 +1,34 @@
-export const affiliate_signin_url = 'https://login.deriv.com/signin.php'
-export const affiliate_signup_url = 'https://login.deriv.com/signup.php'
+const isBrowser = () => typeof window !== 'undefined'
+
+const supported_domains = ['deriv.com', 'deriv.me']
```
Comment (yashim-deriv) [inline src/common/constants.js:3]: Will this cause problems for `localhost` environments?

Comment (ShuvoHabib) [reply src/common/constants.js:3]: For localhost, it will open ''deriv.com'' url

File: src/pages/interim/affiliate/_what-to-know.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ const WhatToKnow = () => {
                 </Text>
                 <Text align="center" size="var(--text-size-m)">
                     <Localize
-                        translate_text="The new affiliate login URL will change to <0>login.deriv.com</0> and you will automatically be redirected to this address on <1>3 June 2020 (Wednesday)</1>."
+                        translate_text={`The new affiliate login URL will change to <0>${login_url}</0> and you will automatically be redirected to this address on <1>3 June 2020 (Wednesday)</1>.`}
```
Comment (sean-binary) [inline src/pages/interim/affiliate/_what-to-know.js:26]: I think you have to use `values` here, e.g. `values={{ login_url }}` then place `<0>{{login_url}}</0>`

File: src/common/constants.js (hunk 1 above)
Comment (ShuvoHabib) [reply src/common/constants.js]: Hi @mustofa-binary , Resolved

File: src/pages/interim/affiliate/_what-to-know.js (hunk 3 above)
Comment (ShuvoHabib) [reply src/pages/interim/affiliate/_what-to-know.js:26]: Hi @sean-binary , do you have any idea, from where I can view this page ?

Comment (sean-binary) [reply src/pages/interim/affiliate/_what-to-know.js]: I'll pm you.

File: src/pages/interim/affiliate/_what-to-know.js [hunk 4]
```diff
@@ -23,7 +23,8 @@ const WhatToKnow = () => {
                 </Text>
                 <Text align="center" size="var(--text-size-m)">
                     <Localize
-                        translate_text="The new affiliate login URL will change to <0>login.deriv.com</0> and you will automatically be redirected to this address on <1>3 June 2020 (Wednesday)</1>."
+                        translate_text="The new affiliate login URL will change to <0>${login_url}</0> and you will automatically be redirected to this address on <1>3 June 2020 (Wednesday)</1>."
```
Comment (sean-binary) [inline src/pages/interim/affiliate/_what-to-know.js:26]:
```suggestion
                        translate_text="The new affiliate login URL will change to <0>{{login_url}}</0> and you will automatically be redirected to this address on <1>3 June 2020 (Wednesday)</1>."
```

**Review by sean-binary — APPROVED** [review]

**Review by mustofa-binary — APPROVED** [review]
